vpr/src/route/parallel_connection_router.h

```diff
@@ -10,6 +10,11 @@
 #include "router_stats.h"
 #include "spatial_route_tree_lookup.h"
 
+#include <fstream>
```

Comment:
Although it may seem silly, this should also be IFDEFed for "PROFILE_HEAP_OCCUPENCY". Its just good practice to only include when you need things.